tests/integration/api/test_user.py

```diff
@@ -339,6 +349,7 @@ def test_list_users(
             "resource_type": admin_user.resource.resource_type,
         },
     }
+    assert results[0]["id"] != system_user.id
```

Comment:
This should not check for all the results instead of just the first one?

Reply:
there is a `len(results) == 1` check above